pkg/datasources/databases.go

```diff
@@ -58,110 +61,26 @@ var databasesSchema = map[string]*schema.Schema{
 		Computed:    true,
 		Description: "Holds the output of SHOW DATABASES.",
 		Elem: &schema.Resource{
-			Schema: map[string]*schema.Schema{
-				"created_on": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"name": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"kind": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"is_transient": {
-					Type:     schema.TypeBool,
-					Computed: true,
-				},
-				"is_default": {
-					Type:     schema.TypeBool,
-					Computed: true,
-				},
-				"is_current": {
-					Type:     schema.TypeBool,
-					Computed: true,
-				},
-				"origin": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"owner": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"comment": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"options": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"retention_time": {
-					Type:     schema.TypeInt,
-					Computed: true,
-				},
-				"resource_group": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"owner_role_type": {
-					Type:     schema.TypeString,
-					Computed: true,
-				},
-				"description": {
-					Type:        schema.TypeList,
-					Computed:    true,
-					Description: "Holds the output of DESCRIBE DATABASE.",
-					Elem: &schema.Resource{
-						Schema: map[string]*schema.Schema{
-							"created_on": {
-								Type:     schema.TypeString,
-								Computed: true,
-							},
-							"name": {
-								Type:     schema.TypeString,
-								Computed: true,
-							},
-							"kind": {
-								Type:     schema.TypeString,
-								Computed: true,
-							},
+			Schema: resources.MergeMaps(
+				schemas.DatabaseShowSchema,
+				map[string]*schema.Schema{
+					"describe_output": {
+						Type:        schema.TypeList,
+						Computed:    true,
+						Description: "Holds the output of DESCRIBE DATABASE.",
+						Elem: &schema.Resource{
+							Schema: schemas.DatabaseDescribeSchema,
```

Comment:
It's inconsistent that `schemas.DatabaseDescribeSchema` is wrapped in `map[string]*schema.Schema`, but `schemas.DatabaseShowSchema` is not.

Reply:
Hmm, they're both represented as `map[string]*schema.Schema`